ambari/README.md

```diff
@@ -24,8 +24,7 @@ start collecting your Ambari performance data. See the [sample ambari.d/conf.yam
 
 ## Data Collected
 
-If host metrics collection is enabled with `collect_host_metrics` this integration collects for every host in every 
-cluster the following system metrics:
+This integration collects metrics for every host in every cluster the following system metrics:
```

Comment:
Why describing the type of metrics collected here and then again in the `###Metrics` section ? I would advocate to remove this part and only keep the one in the Metrics section.